src/App.jsx

```diff
@@ -28,8 +29,17 @@ export default function App() {
 
   return (
     <div className={styles.App}>
-      <Toaster />
-      <Header />
+      <Toaster
+        toastOptions={
+          theme === "dark" && {
+            style: {
+              background: "#333",
+              color: "#fff",
+            },
+          }
+        }
+      />
```

Comment:
That would be so much better.

```jsx
<Toaster toastOptions={toastOptions} />
```

Also if we can give colors as css variable we may not need to add theme condition.